ods_ci/tests/Resources/Files/accelerator/accelerator-profile-3349-2.yaml

```diff
@@ -0,0 +1,17 @@
+# File to provision an Accelerator Profile instance in test:
+#   "Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI"
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: accelerator-profile-3349-2
+  namespace: redhat-ods-applications
```

Comment:
Can you set `namespace` as a parameter (same as you do on the `description` field) so it can be set to
`${APPLICATIONS_NAMESPACE}` ? Same for the other yaml files

Reply:
Sure! Done [f8c06b4](https://github.com/red-hat-data-services/ods-ci/pull/1209/commits/f8c06b4dbff37e37839d68cacc141f21cbf84446)